spec/features/authorization_of_conversation_spec.rb

```diff
@@ -37,6 +46,7 @@
 
       expect(page).to have_content("Chatroom")
       expect(page).to have_css('form.new_message')
+      expect(page).to have_css('form.new_rating')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.